.github/workflows/test.yml

```diff
@@ -42,3 +42,16 @@ jobs:
           license: MPL-2.0
           binary: ${{ steps.build.outputs.binary-path }}
           bin_path: /usr/local/bin
+          file_permissions: 0o027
+
+      - name: dump RPM
+        run: |
+          echo "::group::maybe install rpm" 1>&2
```

Comment:
> maybe install rpm
Is this a group name?

Reply:
Yup, the group is just to hide the output from the package install if it happens, because that's almost never a part of the output we'll want to see.  Click the twisty in the unlikely event you want to see oodles of output from apt 😆 

<img width="721" alt="Screenshot 2024-06-11 at 12 40 00" src="https://github.com/hashicorp/actions-packaging-linux/assets/801275/ebae9bd1-a2fc-407e-8bb4-8a42da1ee410">